deps/build.jl

```diff
@@ -11,7 +11,8 @@ curr_win = "20170502"
 if haskey(ENV, "MXNET_HOME")
   info("MXNET_HOME environment detected: $(ENV["MXNET_HOME"])")
   info("Trying to load existing libmxnet...")
-  lib = Libdl.find_library(["libmxnet", "libmxnet.so"], ["$(ENV["MXNET_HOME"])/lib"])
+  lib = Libdl.find_library("libmxnet.$(Libdl.dlext)",
```

Comment:
I believe under linux you need to continue to use `"libmxnet"` without the `Libdl.ext`

Reply:
seems both work.
I can add it back if you want it.

```
% uname -a
Linux kaladbolg 4.12.8-2-ARCH #1 SMP PREEMPT Fri Aug 18 14:08:02 UTC 2017 x86_64 GNU/Linux
% ls
libmxnet.so*  libmxnet-test.so*
```
```
julia> p = Libdl.find_library(["libmxnet-test.so"], [pwd()])        
"/home/iblis/.julia/v0.5/MXNet/deps/usr/lib/libmxnet-test.so"       

julia> Libdl.dlopen_e(p)          
Ptr{Void} @0x0000000002f4c7b0
```

Reply:
Interesting. Then we need to figure out why Travis is failing.